include/mbgl/ios/MGLAnnotationImage.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
```

Comment:
This file should be audited for nullability. Since neither property should ever be `nil`, surround the header with `
NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` and import MGLTypes.h (to get the Xcode 6.1 compatibility shim).